cli/src/factory_ops/boxing/prov_tasks

```diff
@@ -212,71 +218,106 @@ function boxing_flag {
-    ssh $_node "subscription-manager auto-attach --disable || true"  | tee -a ${LOG_FILE}
+    echo "DEBUG: Running the subscription-manager auto-attach on ${_server}" >> ${LOG_FILE}
+    $ssh_cmd $_pvt_ip "subscription-manager auto-attach --disable || true"  | tee -a ${LOG_FILE}
 
-    echo "Running subscription-manager remove --all on ${_node}" | tee -a ${LOG_FILE}
-    ssh $_node "subscription-manager remove --all || true" | tee -a ${LOG_FILE}
+    echo "DEBUG: Running subscription-manager remove --all on ${_server}" >> ${LOG_FILE}
+    $ssh_cmd $_pvt_ip "subscription-manager remove --all || true" | tee -a ${LOG_FILE}
 
-    echo "Running subscription-manager unregister on ${_node}" | tee -a ${LOG_FILE}
-    ssh $_node "subscription-manager unregister || true" | tee -a ${LOG_FILE}
+    echo "DEBUG: Running subscription-manager unregister on ${_server}" >> ${LOG_FILE}
+    $ssh_cmd $_pvt_ip "subscription-manager unregister || true" | tee -a ${LOG_FILE}
 
-    echo "Running subscription-manager clean on ${_node}" | tee -a ${LOG_FILE}
-    ssh $_node "subscription-manager clean || true" | tee -a ${LOG_FILE}
+    echo "DEBUG: Running subscription-manager clean on ${_server}" >> ${LOG_FILE}
+    $ssh_cmd $_pvt_ip "subscription-manager clean || true" | tee -a ${LOG_FILE}
 
-    echo "Running subscription-manager config --rhsm.manage_repos=0 on ${_node}" | tee -a ${LOG_FILE}
-    ssh $_node "subscription-manager config --rhsm.manage_repos=0" | tee -a ${LOG_FILE}
+    echo "DEBUG: Running subscription-manager config --rhsm.manage_repos=0 on ${_server}" >> ${LOG_FILE}
+    $ssh_cmd $_pvt_ip "subscription-manager config --rhsm.manage_repos=0" | tee -a ${LOG_FILE}
+    echo "Done." | tee -a ${LOG_FILE}
 }
 
 seagate_refs_cleanup()
 {
-    _node_list=("srvnode-1" "srvnode-2")
+    # 1. Check if subscription manager is enabled
+    # 2. Disable and cleanup the subscription
+    # 3. Remove all Seagate internal repos from /etc/yum.repos.d
 
-    for _node in "${_node_list[@]}"; do
+    for i in "${!pvt_ips[@]}"; do
         subscription_enabled=false
-        sub_manager_check "${_node}"
+        sub_manager_check "${pvt_ips[$i]}" "${server_names[$i]}"
         if [[ "$subscription_enabled" == true ]]; then
-            sub_manager_cleanup "$_node"
+            sub_manager_cleanup "${pvt_ips[$i]}" "${server_names[$i]}"
         fi
 
-        echo "Removing following repos from $_node" | tee -a ${LOG_FILE}
-        ssh "$_node" 'grep -lE "seagate.com" /etc/yum.repos.d/*.repo' | tee -a ${LOG_FILE}
-
-        ssh "$_node" 'for file in `grep -lE "seagate.com" /etc/yum.repos.d/*.repo`; do rm -f "$file"; done'
-        echo "Cleaning yum cache on $_node" | tee -a ${LOG_FILE}
-        ssh "$_node" 'yum clean all'
-
-        echo "Done" | tee -a ${LOG_FILE}
+        echo "Removing the repos with Seagate URL from Server A" | tee -a ${LOG_FILE}
+        mkdir -p /opt/seagate/eos-prvsnr/generated_configs/repos_boxing
```

Comment:
is this adjusted for GA?

Reply:
No, just taking the backup, not related to Beta.
FOr GA same logic will be followed - copy repos at this location during boxing.